tests/Makefile

```diff
@@ -24,7 +24,7 @@ PRGDIR  = ../programs
 PYTHON ?= python3
 TESTARTEFACT := versionsTest
 
-DEBUGLEVEL ?= 1
+DEBUGLEVEL ?= 2
```

Comment:
> Set `DEBUGLEVEL=2` by default in `tests/Makefile` to catch compiler errors in `DEBUGLOG()` statements.

There (is ? / used to be ?) a CI test with `DEBUGLEVEL` set to 2 specifically for this purpose (check `DEBUGLOG()` statements).

The question is : 
should checking validity of `DEBUGLOG()` statement be the purpose of a specific CI test,
rather than default for any compilation within `tests/` ?

My concern is that traces from `DEBUGLEVEL = 2` could impact the generated binary layout more than just `assert()`.

Reply:
> There (is ? / used to be ?) a CI test with DEBUGLEVEL set to 2 specifically for this purpose (check DEBUGLOG() statements).

I added this because only Appveyor caught it.

> My concern is that traces from DEBUGLEVEL = 2 could impact the generated binary layout more than just assert().

Does it matter for test cases?

Reply:
> Does it matter for test cases?

I'm not sure.
I was concerned that it could make a sensible performance difference,
reducing the nb of fuzzer tests performed within a given timeframe.

It could also be that the difference is not that large.

Reply:
So I went ahead and tested the performance difference between `DEBUGLEVEL=1` and `DEBUGLEVEL=2` on a given test workload : `time ./fuzzer -v -i1000 -t1 -s7653`

There was a performance impact, but I believe it's small enough, < 5% (time went up from 22.3s to 23.2s).